package.json

```diff
@@ -1,12 +1,16 @@
 {
```

Comment:
more references to bls-ts-key-mgmt in this file

Reply:
where?

Reply:
L21 and a few others